roles/openshift_control_plane/defaults/main.yml

```diff
@@ -156,3 +156,5 @@ l_core_api_list:
 - "security.openshift.io"
 - "template.openshift.io"
 - "user.openshift.io"
+
+openshift_volume_recycler_image: "{{ l_osm_registry_url | regex_replace('${component}' | regex_escape, 'recycler') }}"
```

Comment:
This will still have ${version} in the image name by the time you use it in most cases. Does that need to be evaluated by ansible?

Reply:
@sdodson that is not true.

Reply:
@sdodson we seem to be using same pattern everywhere and I have tested this change on a real cluster. It does seem to replace `$version` with actual version in generated yaml. I can't say for sure if that will happen always (ansible noob here!).